platform/hale-platform.target

```diff
@@ -48,8 +48,8 @@
 		<unit id="de.fhg.igd.equinox.test.feature.feature.group" version="1.2.0.202203220819"/>
 	</location>
 	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
-		<repository location="https://gitlab.wetransform.to/hale/hale-build-support/raw/556667e05ea86da8d32e6aade07489e7872806bc/updatesites/platform"/>
-		<unit id="eu.esdihumboldt.hale.platform.feature.group" version="5.0.0.i202310282027"/>
+		<repository location="https://gitlab.wetransform.to/hale/hale-build-support/raw/b5b31b93703b8f8631bc70fe365b1039e2ac5ec0/updatesites/platform"/>
```

Comment:
Would be great if you use your own [fork](https://gitlab.wetransform.to/hale/hale-build-support/forks) of the repository as discussed for pushing your changes to, until we have a working version. This would prevent too many intermediate binary changes to make it into the main repository and increase its size further. Thanks!